File: 프로그래머스/0/120906. 자릿수 더하기/자릿수 더하기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        
        // n이 0이 될 때까지 반복
        while (n > 0) {
            // n의 마지막 자리 숫자를 더함
            answer += n % 10;
            // n을 10으로 나누어 마지막 자리를 제거
            n /= 10;
        }
        
        return answer;
    }
}

File: 프로그래머스/1/12931. 자릿수 더하기/자릿수 더하기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    while (n > 0) {
        answer += n % 10;  // 마지막 자리 더하기
        n /= 10;           // 마지막 자리 제거
    }
    return answer;
}

